core/threaded/reactor_threaded.c

```diff
@@ -1137,8 +1137,23 @@ int lf_reactor_c_main(int argc, char* argv[]) {
 
     if (process_args(default_argc, default_argv)
             && process_args(argc, argv)) {
-        lf_mutex_lock(&mutex); // Sets start_time
-        initialize();
+        
+        // If _lf_number_of_threads is 0, it means that it was not provided on
```

Comment:
Maybe better to put this in a separate function (and call it inside of `initialize()`).

Reply:
`initialize()` is in reactor_common.c, which is also used in the unthreaded runtime.

Reply:
Ah, you're right; please ignore the latter part of my suggestion.